fiware-region-sanity-tests/commons/fiware_cloud_test_case.py

```diff
@@ -400,6 +400,8 @@ def reset_world_containers(cls, world, suite=False):
         # release resources to ensure a clean world
         for container in list(world['containers']):
             try:
+                for object in cls.swift_operations.get_container(container)[-1]:
```

Comment:
The 'get_container' method can return more than one container? 
Why you try to remove objects only in the last container?